src/quemb/molbe/solver.py

```diff
@@ -893,7 +897,11 @@ def solve_block2(
     # pylint: disable-next=E0611
     from pyscf import dmrgscf  # noqa: PLC0415   # optional module
 
+    if ompnum is not None:
+        dmrgscf.settings.MPIPREFIX = "mpirun -np " + str(ompnum) + " --bind-to none"
```

Comment:
I'm confused about this line. Are you using `ompnum` as the number of OpenMP threads or MPI processes?

Reply:
good catch